src/components/InfoPanel/infoPanel.module.scss

```diff
@@ -5,29 +5,31 @@
   background-color: white;
   display: flex;
   flex-direction: row;
-  justify-content: center;
-  align-items: center;
   position: absolute;
   border: 1px solid black;
   border-radius: 10px;
   z-index: 3000;
-  text-align: center;
-  bottom: -100px;
+  overflow-wrap: break-word;
+  bottom: -70px;
   right: auto;
   @media ($mobile) {
-    bottom: -100px;
+    bottom: -70px;
+    min-height: 120px;
     right: auto;
   }
   @media ($tablet) {
-    bottom: -100px;
+    bottom: -60px;
     right: auto;
   }
 }
 .block {
-  padding: 20px 0;
-  width: 80px;
-  height: 150px;
+  padding: 20px 3px;
```

Comment:
For future projects I would recommend to use padding one on the parent component level to avoid doing the same spacing multiple times. And you should think more about name of the css classes.

Reply:
done